resources/content/schema/admin/admin-auth.json

```diff
@@ -156,9 +239,31 @@
         "publish.transitioningProgress": "cr",
 
         "resourceDefinition" : "r",
+        "resourceDefinition.name" : "r",
+        
+        "registrationToken" : "cr",
+        "registrationToken.state":"r",
+        "registrationToken.registrationUrl" : "r",
+        
+        "register": "cr",
+        "register.key":"cr",
+        "register.secretKey" : "r",
+        "register.accessKey" : "cr",
 
         "schema" : "r",
+        "schema.collectionActions" : "r",
```

Comment:
I'll remove all the fields in the schema then and add this regex in the beginning of the file.